src/docbuilder/metadata.rs

```diff
@@ -53,7 +53,7 @@ pub struct Metadata {
     /// System dependencies.
     ///
     /// Docs.rs is running on a Debian jessie.
-    pub dependencies: Option<Vec<String>>,
+    dependencies: Option<Vec<String>>,
```

Comment:
`Metadata::dependencies` is never used by docs.rs and it's not mentioned in the docs: https://docs.rs/about#metadata
Wouldn't it be cleaner to completely remove this field?
(Can be done in a separate PR ofc)

Any idea what the original intention was of this data? This seems to conflict with the use of `crates-build-env`, since this container already controls all of the available system dependencies...

Reply:
Yeah, so originally docs.rs used a custom sandbox that had a very custom environment, and the idea was we would run `apt-get install <dependencies>` if you had it set. Since we're using docker now it doesn't make sense anymore, I agree we should get rid of it. I actually had someone trying to use it the other day and I had to tell them it wasn't implemented: https://github.com/rust-lang/docs.rs/issues/164